lib/galaxy/jobs/runners/drmaa.py

```diff
@@ -62,6 +54,14 @@ def __init__( self, app, nworkers, **kwargs ):
             log.info( 'Overriding DRMAA_LIBRARY_PATH due to runner plugin parameter: %s', self.runner_params.drmaa_library_path )
             os.environ['DRMAA_LIBRARY_PATH'] = self.runner_params.drmaa_library_path
 
```

Comment:
Maybe adding a comment here explaining why the import should be delayed may prevent the recurrence of this bug.